pkg/diagnostics/v2/client.go

```diff
@@ -108,6 +108,30 @@ func (c *Client) Create() (string, error) {
 	}
 }
 
+// Delete deletes the given cluster bundle.
+func (c *Client) Delete(id string) error {
+	req, err := c.http.NewRequest("DELETE", fmt.Sprintf("%s/%s", baseURL, id), nil)
```

Comment:
How about extract `fmt.Sprintf("%s/%s", baseURL, id)` into a function?
```suggestion
	req, err := c.http.NewRequest("DELETE", bundleUrl(id), nil)
```

Reply:
I'm not convinced we should introduce a level of abstraction here for sth. trivial like this.